src/dgnss_management.c

```diff
@@ -395,6 +395,14 @@ void dgnss_update_ambiguity_state(ambiguity_state_t *s)
   }
 }
 
+/* Interpret raim-related content of baseline function return code. */
+void fill_property_flags(s8 ret, bool fixed, dgnss_baseline_t *solution)
```

Comment:
This looks like a helper function not useful outside of this file.  Make it static and remove from the header.